cloth-diapers/cloth-diapers-basics.md

```diff
@@ -0,0 +1,296 @@
+## Cloth Diapers
+
+There are many sections that need to be written up in this document, but I'll
+start by writing up the critical sections required for basic knowledge
+or those that I have a specific opinion about.
+
+_Disclaimer_: These are my personal opinions and as such, don't come at me if
+yours are different!
+
+However, the plan is to include the following
++ Nomenclature (multiple languages)
++ [Types of Cloth Diapers](#types-of-cloth-diapers)
++ [Materials and Absorbency](#materials-and-absorbency)
++ [Newborn vs One Size](#newborn-vs-one-size)
++ Velcro vs Poppers
++ [Our Choices](#our-choices)
++ Things to Know
++ What causes leaks
+
+## Types of Cloth Diapers
+![image](https://user-images.githubusercontent.com/4185328/169092681-4b44aaef-1127-4937-b906-1b2ddc864821.png)
+
+I don't want to go into all the different types of cloth diaper just yet as
+there are actually a lot of decent resources online already on this, such as
+https://www.thenappygurus.com/types-of-cloth-nappies.html
+
+The whole of the [Nappy Gurus Help & Advice](https://www.thenappygurus.com/help.html)
+section is extremely well put together and very helfpul if you're starting out.
+
+I'm just leaving this diagram here as a reference point to discuss the
```

Comment:
Again, if you are keeping this repo/resource personal to yourself, then "I'm ..." is fine. 
Just checking what your plan is for the writing style. 💭

Reply:
I like the personal writing style. But I'll add a task to go through and update for inclusivity.